.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Please describe your changes -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+## Screenshots (jpeg or gifs if applicable):
+
+## Types of changes
+<!-- What types of changes does your code introduce?  -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->
```

Comment:
The above _Types of Changes_ verbiage should match that of the existing [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#workflow), `add`, `try`, `update` etc